### DISCUSSION TITLE
Quarto - Quarto for Academics

### DISCUSSION THREAD

**Discussion by giscus** [quarto-dev/quarto-cli#11417] [Blog]
# Quarto - Quarto for Academics

A video highlighting some of Quarto’s features that are especially useful for academics, as educators and as researchers.

https://quarto.org/docs/blog/posts/2023-05-22-quarto-for-academics/

**Comment by drfurtado** [comment]
Excellent! I am slowly moving my academic life into [Quarto](https://quarto.org).

**Comment by higgi13425** [comment]
Hoping for options for creating and printing meeting posters (US 3ft tall by 6ft wide, europe roughly 4ft tall by 3ft wide). Does Typst make this possible?
